packages/commonwealth/client/scripts/views/components/NewThreadForm/helpers/helpers.ts

```diff
@@ -25,13 +26,21 @@ export const checkNewThreadErrors = (
   }
 };
 
-export const checkIsTopicInContest = (data: Contest[], topicId?: number) => {
+export const checkIsTopicInContest = (
+  data: Contest[],
+  topicId?: number,
+  checkOnlyActiveContest = false,
+) => {
   if (!topicId) {
     return false;
   }
 
-  return (data || []).some(
-    (item) =>
-      item?.topics && item?.topics.some((topic) => topic?.id === topicId),
-  );
+  return (data || [])
+    .filter((item) =>
+      checkOnlyActiveContest ? isContestActive({ contest: item }) : true,
+    )
+    .some(
+      (item) =>
+        item?.topics && item?.topics.some((topic) => topic?.id === topicId),
```

Comment:
```diff
- item?.topics && item?.topics.some(
+ item?.topics && item?.topics?.some?.(
```

Reply:
Not necessary because `item?.topics` will check if there are topics present to filter over.